### ISSUE TITLE
[Question]: Floating Toolbars

### ISSUE THREAD

**Issue by PWagner1** [Krypton-Suite/Standard-Toolkit#1136]
Is it time to include the floating toolbars, which are currently in the `Extended-Toolkit` into the `Standard-Toolkit`?

**Comment by Smurf-IV** [comment]
When did they become part of the Winforms standard ?
Something to consider once V8 is in stable and is master

**Comment by Wagnerp** [comment]
> When did they become part of the Winforms standard ?

Not a part of vanilla WinForms, but still a good feature to have, especially for MDI applications

**Comment by Smurf-IV** [comment]
@Wagnerp Does the Ext TK have an example for the MDI Floating bars ?
If not please create, so that it and this feature can be moved at the same time.

**Comment by Wagnerp** [comment]
@Smurf-IV Yes it should be part of the example app, if not there is an animation here

![](https://github.com/Krypton-Suite/Documentation/blob/main/Assets/Extended-Toolkit/FloatableToolStrips.gif)

**Comment by Smurf-IV** [comment]
I see the animation, but I do not understand the "Need" for them to be floated outside the parent application.
It's an "Uncommon" use case, so it they should stay in EXT.
The VB IDE is no longer "Cool"
Convince me otherwise ....

**Comment by Wagnerp** [comment]
Closing, they're part of Extended.

**Comment by PWagner1** [comment]
Reopening, adding as part of `Krypton.Utilities`
